### ISSUE TITLE
Reports That All A HREF Links in Snippets Are No Longer Working

### ISSUE THREAD

**Issue by bensternthal** [mozmeao/snippets-service#528]
Reported via slack from Michelle Rae:

> All running snippets with a link have dead links. the code is there when you mouseover. It's not working for anybody on any channel. I can't test links in Preview because links in previewing haven't worked for a couple of days.

I am able to verify that something is broken. Testing here it looks like the hrefs are there but clicking does not work. I am suspicious of a [recent PR.](https://github.com/mozmeao/snippets-service/pull/519)

**Comment by k88hudson** [comment]
I took a quick look at this, I'm thinking it might be this line here: 
https://github.com/mozmeao/snippets-service/pull/519/files#diff-050fa7fbdf0fa60913c8fc36985fa302R567

Re-triggering the click would run this through the handler a second time and cause `e.preventDefault` again no?

**Comment by bensternthal** [comment]
I agree.

OK will need to wait until Monday when we can do a push. I spoke with the snippets content team about which content can run from now until monday. Also emailed Giorgos so he knows to work on this first thing.

**Comment by glogiotatidis** [comment]
Fixed [here](https://github.com/mozmeao/snippets-service/pull/529) and pushed to prod.

tl;dr introducing a new special link to open the ham menu, complicated the metric collection and broke non-newtab 'normal'  links  by calling `preventDefault` always. 


Thanks for the help!
